src/lib/utils/analytics.ts

```diff
@@ -1,11 +1,13 @@
 import posthog from "posthog-js";
 
-function initiateAnalytics () {
+function initiateAnalytics() {
   // eslint-disable-next-line
```

Comment:
```suggestion
function initiateAnalytics() {
  // eslint-disable-next-line
```

Moving this down to the 'api_host' line should fix the problem. Sorry about that!

Reply:
Why move the disable line down? Why not change the var name to make it camel case?

Reply:
Also, this is an interesting situation. I didn't change the var name. That already existed in the codebase but the error is popping up because the formatting changed from prettier. Prettier should help to resolve these situations in the future.

Reply:
Oh so the attribute in the object is specifically called api_host from the posthog package so yeah. 🤷